### DISCUSSION TITLE
十月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#427] [通知]
## 🚀 功能

- 提供基本的XBOX操作支持 @Richasy (#410)

## 🐛 Bug 修复

- 修复互动视频选项过长时的UI问题 @Richasy (#423)
- 在执行操作后自动更新视频互动信息 @Richasy (#422)
- 修复弹幕字体不生效的问题 @Richasy (#421)
- 添加迷你播放控制器 @Richasy (#420)
- 修复窄视图下视频不显示的问题 @Richasy (#415)
- 修复PGC内容中错误的用户显示 @Richasy (#414)
- 尝试修复网络视图滚动问题 @Richasy (#413)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2110.2.0'>十月更新</a>.</em>

**Comment by DeepslateQAQ** [comment]
UWP版的BILIBILI真的是太香了

**Comment by Sherlock-haoran** [comment]
NICE_**!**_

**Comment by PRO-2684** [comment]
好！

**Comment by W-H-Chen** [comment]
好棒的uwp！！但在观看多集数的时候，自动连续播放哪里可以开？然后分集没看到滚动条。。。
